feature-nft-impl/src/main/java/io/novafoundation/nova/feature_nft_impl/data/source/providers/uniques/UniquesNftProvider.kt

```diff
@@ -48,7 +56,12 @@ class UniquesNftProvider(
     private val ipfsApi: IpfsApi,
 ) : NftProvider {
 
-    override suspend fun initialNftsSync(chain: Chain, metaAccount: MetaAccount, forceOverwrite: Boolean) {
+    override suspend fun initialNftsSync(
+        chain: Chain,
+        metaAccount: MetaAccount,
+        forceOverwrite: Boolean,
+        at: BlockHash?
+    ) {
         val accountId = metaAccount.accountIdIn(chain) ?: return
 
         val newNfts = remoteStorage.query(chain.id) {
```

Comment:
`at` seems to be unused here. Have you meant passing it to `remoteStorage.query(chain.id, at=at)`?

Reply:
Done